__tests__/server.test.js

```diff
@@ -71,4 +71,54 @@ describe("GET /api/articles/:article_id", ()=>{
             expect(body.msg).toBe("Bad request")
         })
     })
+})
+
+describe("PATCH: /api/articles/:article_id", ()=>{
+    test("200: Returned updated article",()=>{
+        const incVotes = {inc_votes: 1}
+        return request(app)
+        .patch("/api/articles/1")
+        .send(incVotes)
+        .expect(200)
+        .then(({body: {article}})=> {
+            expect(article).toMatchObject({
+                article_id: 1,
+                title: "Living in the shadow of a great man",
+                topic: "mitch",
+                author: "butter_bridge",
+                body: "I find this existence challenging",
+                created_at: expect.any(String),
+                votes: 101,
+                article_img_url:
+                  "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
+              })
+        })
+    })
+    test("400: Incorrect Data type", ()=>{
+        const incVotes = {inc_votes: 'one'}
+        return request(app)
+        .patch("/api/articles/1")
+        .send(incVotes)
+        .expect(400)
+        .then(({body})=> {
+            expect(body.msg).toBe("Bad request")
+        })
+    })
+    test("400: No data Given", ()=>{
+        return request(app)
+        .patch("/api/articles/1")
+        .expect(400)
+        .then(({body})=>{
+            expect(body.msg).toBe("Bad request")
+        })
+    })
+    test("404: No article Found", ()=>{
+        const incVotes = {inc_votes: 1}
+        return request(app)
+        .patch("/api/articles/999")
+        .send(incVotes)
+        .then(({body})=>{
+            expect(body.msg).toBe("Article not found")
+        })

```

Comment:
One more test to go here, what about patching to a bad url? eg /api/articles/banana what would/should happen?